MANIFEST.in

```diff
@@ -1,4 +1,5 @@
 include *.md *.txt
 include MANIFEST.in
+recursive-include benchmarks *
```

Comment:
Why include benchmark scripts in the sdist? It doesn't get installed right? Honestly I wouldn't even include tests/docs/txt either

Reply:
it's only in sdist, not in the build dist which only has a few files.

Reply:
But what's the point of including these in the sdist?

Reply:
Up to you. Personally I try to keep the sdist limited to the smallest possible collection of files needed to build the bdist and use the GitHub tarball as the authoritative collection of all files.

Reply:
I'll keep it in sdist for now as a matter of continuity, and that it's a really small file that someone may find useful.